contracts/0.4.24/template/LidoTemplate.sol

```diff
@@ -653,12 +652,10 @@ contract LidoTemplate is IsContract {
         perms[0] = _state.lido.PAUSE_ROLE();
         perms[1] = _state.lido.MANAGE_FEE();
         perms[2] = _state.lido.MANAGE_WITHDRAWAL_KEY();
-        perms[3] = _state.lido.SET_ORACLE();
+        perms[3] = _state.lido.MANAGE_PROTOCOL_CONTRACTS_ROLE();
```

Comment:
need to cover the new roles in LidoTemplate.sol: `RESUME_ROLE`, `STAKING_PAUSE_ROLE`, etc
and probably in `protocol-levers.md`, `12-check-dao.js`, `deploy.js` (search by `PAUSE_ROLE` occurrences)

Reply:
Maybe is not an issue cause we decided to minimize the assigned roles, wdyt?